src/Services/UserService.php

```diff
@@ -5,28 +5,38 @@
 use Psr\Http\Message\RequestInterface;
 use QuizApp\Entities\User;
 use ReallyOrm\Repository\RepositoryManagerInterface;
+use ReallyOrm\Test\Repository\RepositoryManager;
 
 class UserService extends AbstractService
 {
+    /**
+     * @var RepositoryManager
+     */
     private $repositoryManager;
 
     public function setRepositoryManager(RepositoryManagerInterface $repositoryManager)
     {
         $this->repositoryManager = $repositoryManager;
     }
 
-    public function getAll(RequestInterface $request)
+    public function getAll(int $page)
     {
-        $page = $request->getRequestParameters();
-        $from = ($page['page'] - 1) * 10;
+        $offset = ($page - 1) * 10;
```

Comment:
you assume the number of items per page is 10. at least make it a constant on the controller.

maybe in the future you want the front-end to control how many items per page, same as how it controls the page number